server/controllers/parties.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import parties from '../database/dummydbParties';
+
+const app = express();
+app.use(express.json());
+
+class Parties {
```

Comment:
Your controller file name should be in Pascal casing and singular with controller added to it, this also should apply to the class naming in the file.

Reply:
Does this apply to all other file names? or just for the controllers?